src/Models/Traits/Product/Searchable.php

```diff
@@ -119,4 +119,27 @@ public function withCategories(array $data): array
 
         return $data;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getIndexMappings(): ?array
+    {
+        return [
```

Comment:
Should we merge this with the mapping Eventy filter?
- https://docs.rapidez.io/3.x/package-development.html#eventy-filters
- https://github.com/search?q=org%3Arapidez+index.product.mapping+language%3APHP&type=code&l=PHP

Reply:
Yes, added (and also for the settings).